charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-init-script.yaml

```
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-proxy-init-script
+  namespace: kube-system
+data:
+  init.sh: |
+    #!/bin/sh
+    OLD_KUBE_PROXY_MODE=$(cat /kube-proxy-mode);
```

Semicolons are not required in shell scripts at the end of the lines.
```suggestion
    OLD_KUBE_PROXY_MODE=$(cat /kube-proxy-mode)
```